Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-slim-buster as base
+RUN apt-get update; apt-get install curl -y
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="/root/.local/bin:$PATH"
+WORKDIR /todo_app
+COPY pyproject.toml /todo_app/pyproject.toml
+COPY poetry.lock /todo_app/poetry.lock
```

Comment:
Really minor, but since you're copying them together into the working directory we can quite easily simplify this if we want into either:
```
COPY pyproject.toml poetry.lock /todo_app/
```

```
COPY pyproject.toml poetry.lock ./
```

I don't necessarily prefer these since the syntax is a bit less obvious, but worth being aware of